content/docs/start/visualization.md

```diff
@@ -0,0 +1,113 @@
+# Visualization with Plots
+
+In this section, we'll add visualization to the [`example-dvc-experiments`][ede]
+project. If you would like to try these yourself, please refer to the project
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+### Creating plots from text with DVC
+
+A useful plot to show the classification performance is the confusion matrix. In
+order to produce it, DVC expects a CSV **plot file** in the form:
+
+```csv
+actual,predicted
+0,0
+0,2
+...
+```
+
+> We added a [loop] comparing the results to generate this file from the
+> predictions.
+
+[loop]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L123
+
+After running the experiment with `dvc exp run`, DVC will produce
+`plots/confusion.csv` file presenting the classifier performance and open it in
+the browser:
+
+```dvc
+$ dvc plots show plots/confusion.csv --template confusion \
+                                     -x actual -y predicted
+file:///Users/.../example-dvc-experiments/plots/confusion.json.html
+```
+
+![confusion matrix](/img/start_visualization_confusion1.png)
+
+### Using plots generated by scripts
+
+Let's produce another plot to see misclassified examples from each class. This
+procedure generates the misclassification examples from the validation data and
+arranges them into a _confusion table_ that shows the correct label, and
+misclassification sample. The code to generate an image from a set of training
+images is omitted here but you can find the code in [the example
+project.][misclassified-example-code]
```

Comment:
I'm trying to simplify this explanation because atm it's very hard to read or understand TBH. How about

```suggestion
Let's produce another plot to see misclassified examples from each class. This
uses validation data, arranging it into a custom _confusion table_ that shows
the wrong label and sample image matches.

> You can find the code in [the example project][misclassified-example-code].
```

> I'm not sure I got the logic exactly right there, please adjust as needed.

Reply:
However, I find it a bit confusing to use a custom image plot example that is ALSO a confusion matrix (but not one generated with the confusion plot template as before). Probably too much to ask at this point but consider using some other kind of plot (or mark this as a possible improvement for the future).